applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_geometry_utilities.cpp

```diff
@@ -0,0 +1,259 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_geometries/line_interface_geometry.h"
+#include "custom_utilities/geometry_utilities.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+
+#include <boost/numeric/ublas/assignment.hpp>
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(GeometryUtilities_RotationMatrixForHorizontalInterfaceIsIdentityMatrix,
```

Comment:
What about being more precise about the kind of line interface (2+2), since then we could have a similar test case name for the horizontal 3+3 line interface (further down)?
```suggestion
KRATOS_TEST_CASE_IN_SUITE(GeometryUtilities_RotationMatrixForHorizontal2Plus2LineInterfaceIsIdentityMatrix,
```